lib/transaction/input/multisigscripthash.js

```diff
@@ -1,29 +1,25 @@
 'use strict';
 
-var _ = require('lodash');
-var inherits = require('inherits');
-var Input = require('./input');
-var Output = require('../output');
-var $ = require('../../util/preconditions');
-
-var Script = require('../../script');
-var Signature = require('../../crypto/signature');
-var Sighash = require('../sighash');
-var PublicKey = require('../../publickey');
-var BufferUtil = require('../../util/buffer');
-var TransactionSignature = require('../signature');
+const _ = require('lodash');
+const inherits = require('inherits');
+const Input = require('./input');
+const Output = require('../output');
+const $ = require('../../util/preconditions');
+const Script = require('../../script');
+const Signature = require('../../crypto/signature');
+const Sighash = require('../sighash');
+const BufferUtil = require('../../util/buffer');
+const TransactionSignature = require('../signature');
 
 /**
  * @constructor
  */
 function MultiSigScriptHashInput(input, pubkeys, threshold, signatures) {
```

Comment:
Do we want add doc description of this class?

Reply:
Same as above